modules/exploits/linux/http/librenms_addhost_cmd_inject.rb

```diff
@@ -0,0 +1,160 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
```

Comment:
I bet this is better than NormalRanking. How about a bump? Like... **EXCELLENT**